README.md

```diff
@@ -204,9 +204,12 @@ These instructions are tested on Linux Ubuntu 22.04 and should also work on clos
       apt install -y \
         build-essential \
         curl \
+        gir1.2-gtk-4.0
```

Comment:
What is driving the dependency on gir-gtk and libcairo2?

Reply:
The top level dependency is PyGObject, which I think was for doing some light GUI stuff. We might not need it anymore. If so, libgirepository1.0-dev can be dropped as well.